src/sumoLogger.js

```diff
@@ -27,44 +29,52 @@ function setConfig(opts) {
     session: SESSION_KEY + (opts.sessionKey ? opts.sessionKey : getUUID()),
     onSuccess: opts.onSuccess || false,
     onError: opts.onError || false
-  };
+  }
 }
 
 function sendLogs() {
   if (currentLogs.length === 0) {
     return;
   }
-  var tempCategory = '';
-
   try {
     var headers = {'Content-Type': 'application/json'};
     if (currentConfig.sourceName !== '') {
-      _.extend(headers, {'X-Sumo-Name': currentConfig.sourceName});
+      assignIn(headers, {'X-Sumo-Name': currentConfig.sourceName});
     }
     if (currentConfig.sourceCategory !== '') {
-      _.extend(headers, {'X-Sumo-Category': currentConfig.sourceCategory});
+      assignIn(headers, {'X-Sumo-Category': currentConfig.sourceCategory});
     }
     if (currentConfig.hostName !== '') {
-      _.extend(headers, {'X-Sumo-Host': currentConfig.hostName});
+      assignIn(headers, {'X-Sumo-Host': currentConfig.hostName});
     }
 
-    request({
-      method: 'POST',
-      url: currentConfig.endpoint,
+    axios({
```

Comment:
@billsaysthis This superficially looks similar to how `request` was being used, but I don't think `sumo` handles the body the same way. That's kind of a black box for me.

Relevant docs -> https://github.com/axios/axios#request-config

```javascript
// `data` is the data to be sent as the request body
// Only applicable for request methods 'PUT', 'POST', and 'PATCH'
// When no `transformRequest` is set, must be of one of the following types:
// - string, plain object, ArrayBuffer, ArrayBufferView, URLSearchParams
// - Browser only: FormData, File, Blob
// - Node only: Stream, Buffer
data: {
  firstName: 'Fred'
}
```

Reply:
Looking through `request`, there's some type-ish checking with `Array.isArray()`

Reply:
We have an internal hackathon in three weeks, I will block some time to verify this and https://github.com/SumoLogic/js-logging-sdk/pull/8 during that week.